src/linker/Linker.Steps/BodySubstituterStep.cs

```diff
@@ -52,13 +48,11 @@ bool ShouldProcessSubstitutions (XPathNavigator nav)
 
 			var value = GetAttribute (nav, "featurevalue");
 			if (string.IsNullOrEmpty (value)) {
-				Context.LogMessage (MessageContainer.CreateErrorMessage ($"Feature {feature} does not specify a \"featurevalue\" attribute", 1001));
```

Comment:
Is there a reason we need to throw here? I'd prefer to recover from this error if possible - same with the other one in this file.